third_party/rust/adss/v0_2/README.chromium

```diff
@@ -5,5 +5,5 @@ Version: 0.2.2
 Security Critical: yes
 Shipped: yes
 License: Mozilla Public License 2.0
```

Comment:
(same here)

Reply:
License names are all auto-generated, but I can put overrides into `brave/third_party/rust/chromium_crates_io/gnrt_config.toml`

Reply:
Ah, what's generating them, is it based on the metadata inside the crates?

Reply:
If that's the case, then let's leave as-is and you can ignore my suggestions below.

We're not currently making use of this field for automatic license checking and we can always add these overrides later when we do need them to be more standardised.